troi/patches/periodic_jams.py

```diff
@@ -100,7 +100,7 @@ def create(self, inputs):
         else:
             jam_types = jam_type.lower()
```

Comment:
typo? should be jam_type = ?

Reply:
ha, unrelated to this PR, but fixed. thanks!